CodeEditModules/Modules/Git/src/Clone/GitCloneView.swift

```diff
@@ -208,7 +252,7 @@ extension GitCloneView {
         // Check if repo has only one branch, and if so, don't show the checkout page
         do {
             let branches = try GitClient.default(directoryURL: dirUrl,
```

Comment:
we should probably use the instance of gitclient we already have instead of creating a new one